vscode/README.md

```diff
@@ -1,29 +1,24 @@
 # Q# Language Support for VS Code
 
-This VS Code extension contains:
+_This extension is an unstable product that is still under active development. We do not recommend relying on this in production environments._
 
-- The TextMate grammar and language configuration for Q#. This enables syntax highlighting
-  and basic syntactic features (brace matching, etc.).
-- Q# cell support in Jupyter Notebooks. The extension will detect `%%qsharp` magic cells
-  and automatically update the cell language to Q#.
-- Error checking in Q# source files.
-- Breakpoint debugging and script execution for Q# source files.
+This extension brings rich Q# language support to VSCode. Develop, build, and run your Q# code from VSCode either locally on simulators, or by submitting a job to Azure Quantum.
```

Comment:
"VSCode" spelling/spacing is inconsistent (e.g. see the title). Please use the space everywhere, e.g. "VS Code".